en/protocol/parameter.md

```diff
@@ -1,7 +1,64 @@
 # Parameter Protocol
 
+> **Caution** This content has not been reviewed since being ported from the old website (and may be out of date). Updates/re-validation welcome!
+
 The parameter protocol is used to exchange key system settings and guarantees delivery.
 
+The onboard parameter interface allows to read and write parameters (e.g. PID gains) into the current memory (RAM) and also into permanent storage.
```

Comment:
Is this useful to say?

Reply:
This used to be the case but it seems these days it's not used anymore. Everything is saved.